### PR TITLE
enrich README with information about case sensitivity

### PR DESCRIPTION
**Pull request by PHaroZ**
refs #1200

### CODE REVIEW

File: mysql/README.md [hunk 1]
```diff
@@ -3,3 +3,9 @@
 An ANTLR4 grammar for MySQL based on version 5.6 and 5.7 of 
 [http://dev.mysql.com/doc/refman/5.6/en/](http://dev.mysql.com/doc/refman/5.6/en/).
 [http://dev.mysql.com/doc/refman/5.7/en/](http://dev.mysql.com/doc/refman/5.7/en/).
+
+## Usage, important note
+As SQL grammar are normally not case sensitive but this grammar implementation is, you must use a custom [character stream](https://github.com/antlr/antlr4/blob/master/runtime/Java/src/org/antlr/v4/runtime/CharStream.java) that converts all characters to uppercase before sending them to the lexer.
```
Comment (KvanTTT) [inline mysql/README.md]: A small issue: please always insert line breaks after headers.

Comment (KvanTTT) [inline mysql/README.md]: A small issue: please always insert line breaks after headers.

Comment (PHaroZ) [reply mysql/README.md]: Sorry but i am not sure to understand which headers you are talking about : do you mean I should add a blank line between lines 7 and 8 ?

Comment (KvanTTT) [reply mysql/README.md]: Yes, I mean that. Markdown header starts with `#` character.

Comment (PHaroZ) [reply mysql/README.md]: done

Comment (KvanTTT) [reply mysql/README.md]: Thank!

**Pull request merged**